ynr/apps/elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -34,7 +33,5 @@ def handle(self, *args, **options):
             if errors:
                 for error in errors:
                     print(error)
-                msg = "The import broke the constraints listed above; " \
-                      "rolling back the transaction..."
+                msg = "The import broke the constraints listed above; " "rolling back the transaction..."
```

Comment:
I can see why this is the output of running this, but in isolation if I come across this line of code in future it reads quite strangely. Do you think its worth attempting to detect odd stuff like this or just accept that there will be some and deal with it as we find it?

Reply:
Oh that is a bit odd! I'd have thought it would have fixed that. I reckon it's worth just fixing them as we find them…